src/ostorlab/runtimes/local/runtime.py

```diff
@@ -519,7 +521,21 @@ def _list_agent_services(self):
 
     def _start_tracker_agent(self):
         """Start the tracker agent to handle the scan lifecycle."""
-        tracker_agent_settings = definitions.AgentSettings(key=TRACKER_AGENT_DEFAULT)
+        tracker_agent_settings = definitions.AgentSettings(
+            key=TRACKER_AGENT_DEFAULT,
+        )
+
+        if self.timeout is not None:
+            tracker_agent_settings.args.extend(
+                [
+                    utils_definitions.Arg(
+                        name="scan_done_timeout_sec", type="number", value=self.timeout
+                    ),
+                    utils_definitions.Arg(
+                        name="postscane_done_timeout_sec", type="number", value=0
```

Comment:
Why do we set this one to 0?

Reply:
But the purpose of the post scan done is that we give time to the agent to finish processing the last tasks (flush vulnerabilities) so setting to 0 will break that.